indico/htdocs/sass/modules/abstracts/_email_templates.scss

```diff
@@ -55,16 +55,73 @@
     }
 }
 
-.email-template-list {
-    list-style-type: none;
-    padding-left: 0;
-    width: 800px;
+#email-template-manager {
+    max-width: 800px;
+
+    .email-template-wrapper {
+        padding: 0 0 1em 1em;
+
+        .explanation {
+            color: $light-black;
+        }
+
+        .email-templates {
+            width: 800px;
+            max-height: 800px;
+            overflow: auto;
+
+            > ul {
+                list-style-type: none;
+                padding-left: 0;
+                padding-right: 20px;
+
+                > li {
+                    width: 775px;
+                }
+            }
+        }
+
+        .email-preview {
+            display: none;
+
+            .email-subject {
+                padding-bottom: 10px;
+            }
+
+            .email-body {
+                color: $light-black;
+            }
+        }
+
+        .regform-section-sortable-placeholder {
+            @extend .i-box;
+            @include border-all($style: dashed);
+            @include box-shadow(none);
+
+            background-color: $light-gray;
+        }
+
+        .email-preview-btn {
+            padding-right: 5px;
+            color: $link;
+        }
 
-    .email-template {
-        margin-bottom: 10px;
+        .email-preview-btn:hover {
```

Comment:
Nothing wrong with this selector. It is sometimes clearer to un-nest selectors. Just letting you know, you can also do it like this:

```scss
.email-preview-btn {
    // ...

    &:hover {
        // ...
    }
}
```